features/dist-archive.feature

```diff
@@ -203,6 +203,114 @@ Feature: Generate a distribution archive of a project
       | zip     | zip       | unzip     |
       | targz   | tar.gz    | tar -zxvf |
 
+  Scenario Outline: Ignores files specified with absolute path and not similarly named files
```

Comment:
In other test we  are using `Scenario:` instead of `Scenario Outline:`

Reply:
`Scenario Outline:` is used where there is an `Examples:` table at the end of the test, like a PHPUnit `@dataProvider`. I'm pretty sure it's correct here.

https://docs.behat.org/en/latest/user_guide/writing_scenarios.html#scenario-outlines

Reply:
Yep, `Scenario Outline:` lets you run multiple variables against the same scenario.